website/views/shared/modals/modify-inventory.jade

```diff
@@ -44,7 +46,7 @@ script(type='text/ng-template', id='modals/modify-inventory.html')
             hr
 
             ul.list-group
-              li.list-group-item(ng-repeat="item in Content.special" ng-init="inv.special[item.key] = user.items.special[item.key]")
+              li.list-group-item(ng-repeat="item in Content.special" ng-init="inv.special[item.key] = user.items.special[item.key]" ng-if="item.value === 15")
```

Comment:
Hmmm... I'm not a fan of hard coding the value... but since it's a debug menu, if it breaks in the future, it's not the end of the world.